px-breadcrumbs.es6.js

```diff
@@ -72,224 +64,137 @@
        */
       clickOnlyMode: {
         type: Boolean,
-        value: false
+        value: false,
+        observer: '_getBreadcrumbsObj'
       },
-      filterMode: {
+      searchMode: {
         type: Boolean,
         value: false
       },
-      _filterString: {
-        type: String,
-        value: ''
-      },
       _hideNoResultLi: {
         type: Boolean,
         value: true
+      },
+      _breadcrumbsObj: {
+        type: Object
       }
     },
-    behaviors: [Polymer.IronResizableBehavior],
+
     listeners: {
-      'iron-resize': '_onResize'
+      'iron-resize': '_onResize',
+      'px-app-asset-selected': '_onSelect',
+      'px-dropdown-selection-changed': '_dropdownTap',
+      'px-app-asset-graph-created': '_getBreadcrumbsObj'
     },
     observers: [
-      '_calculatePath(_selectedItem)', 
-      '_rebuildBreadcrumbsDisplayOptions(_selectedItemPath, _ulWidth)',
-      'prepareData(breadcrumbData)'
-      ],
+      '_getDisplayMode(_ulWidth, _selectedItemPath, _breadcrumbsObj)'
+    ],
     attached() {
-      document.addEventListener('click', this._onCaptureClick.bind(this));
+      this.fire('iron-resize');
```

Comment:
Something must not have been working while I was developing and that helped it, but it seems to be working without it now.